INDEX

```diff
@@ -87,6 +103,13 @@
     "name": "Homebridge",
     "offical": false
   },
+  "hoobs": {
+    "MANIFEST": "hoobs.json",
+    "description": "HOOBS a Homebridge stack with a interface that simplifies configuration and installing plugins. This adds a process wrapper for Homebridge. It also adds an API that can be used as an endpoint for an application.",
+    "icon": "https://icons.freenas.org/community-icons/hoobs.png",
+    "name": "HOOBS",
+    "offical": false
```

Comment:
We are missing `primary_pkg` attribute here, it can be `null`. Also I think we should default to `null` in the first one as well ( we try to do that everywhere where primary pkg has not been specified )